From private repo

v-alje opened the pull request.

PRmerger commented:
@alkohli : Thanks for your contribution to the Azure documentation! The author, @alkohli, has been notified to review your proposed change.

PRmerger commented:
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.

It�s easy to access the private repo version of your articles in GitHub. Just add �pr to the URL, as follows: 

Public repo URL: https://github.com/Azure/azure-content/blob/master/articles/azure-portal-overview.md

Private repo URL: https://github.com/Azure/azure-content-pr/blob/master/articles/azure-portal-overview.md

We've closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, see the contributor guide instructions.

SnehaGunda committed:
Adding new files

SnehaGunda committed:
Adding QS portal article

SnehaGunda committed:
Adding metadata

SnehaGunda committed:
Adding PS quickstart

SnehaGunda committed:
Acrolinx fixes

SnehaGunda committed:
Acrolinx fixes

SnehaGunda committed:
Updating cleanup resources

SnehaGunda committed:
Updating intent statement

SnehaGunda committed:
Updating intent statement

mrbullwinkle committed:
Cohorts draft 001

mrbullwinkle committed:
fix link

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb_03_29_2018_cohorts

SnehaGunda committed:
Title update

SnehaGunda committed:
Minor fix

SnehaGunda committed:
Minor updates

SnehaGunda committed:
Merge branch 'ASAQS' of https://github.com/SnehaGunda/azure-docs-pr into ASAQS

v-viinde committed:
USER STORY 1226462 SaaS App Tutorial: Ziflow

v-viinde committed:
Updated

v-viinde committed:
Updated

carolinacmoravia committed:
L15680: Extra blank space in reference

Hello, @ggailey777, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

<Provide further explanation here as needed>  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

v-viinde committed:
acrolinx score updated

v-nagta committed:
USER STORY 1229999 SaaS App Tutorial: DigiCert Tutorial update

Commit:
Added start and stop articles

Commit:
Fixed syntax issues

thedanfernandez committed:
Update Home.md

Updated to remove link tags for H2's (in markdown as "##") based on customer feedback that these looked liked links.

SnehaGunda committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ASAQS

barbkess committed:
reorganize tiers for gen2

barbkess committed:
fix warning

barbkess committed:
fix warning

SnehaGunda committed:
Updating per CFR feedback

SnehaGunda committed:
Updating per CFR feedback

v-nagta committed:
USER STORY 1224167 SaaS App Tutorial: AppNeta Performance Monitor

CarlRabeler committed:
[sql db] jovan feedback

CarlRabeler committed:
Merge branch 'master' of https://github.com/CarlRabeler/azure-docs-pr

mrbullwinkle committed:
draft 002

CarlRabeler committed:
more

mrbullwinkle committed:
shrink gif for performance

mrbullwinkle committed:
formatting build issue numbered list

barbkess committed:
updates

SnehaGunda committed:
Adding screenshots to create SA

SnehaGunda committed:
updating screendhot

CarlRabeler committed:
more

barbkess committed:
remove performance tiers

CarlRabeler committed:
more

CarlRabeler committed:
typo

CarlRabeler committed:
links

mrbullwinkle committed:
update images

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into gen2

mrbullwinkle committed:
fix numbered formating due to render issue

SnehaGunda committed:
Format fixes

mrbullwinkle committed:
final edits

v-nagta committed:
USER STORY 1226527 SaaS App Tutorial: mindWireless

genlin committed:
add more info about check tcp port

genlin committed:
change a wording

mrbullwinkle committed:
update TOC to add cohorts doc

CarlRabeler committed:
more

cephalin committed:
upgrade to core 2.0

CarlRabeler committed:
more

cynthn committed:
Update storage-about-vhds-and-disks-windows-and-linux.md

https://github.com/MicrosoftDocs/azure-docs/issues/7044

cynthn committed:
Update storage-about-vhds-and-disks-windows-and-linux.md

CarlRabeler committed:
more

Commit:
Updated start and stop for asdk

Commit:
acrolinx fixes

Commit:
fixed syntax issues

Commit:
fixed syntax issues

Commit:
fixed syntax issues

Commit:
swapping art

Commit:
Fixing desc

CarlRabeler committed:
typo

Commit:
fixed syntax issues

barbkess committed:
fix merge conflict

Commit:
fixed syntax issues

barbkess committed:
fix links

barbkess committed:
acrolinx fixes

CarlRabeler committed:
azure

Commit:
Updates for sign in using MSA accounts.

rolyon committed:
Fixed links for new role-based-control folder

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-link-fix

rwike77 committed:
adding stub article

SaurabhSharma-MSFT committed:
(ForumsTeam) Modified the guid example MicrosoftDocs/azure-docs#6967

Guid in the example has 31 characters whereas it should be of 32 characters.

SnehaGunda committed:
merging with master

Commit:
Fixing links and syntax

sethmanheim committed:
Fixing incorrect arm option

barbkess committed:
fix links

barbkess committed:
fix merge conflict

Commit:
merge conflict

Commit:
Fixes

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aas0411

sethmanheim committed:
Updates

mattbriggs committed:
adding trademark

sethmanheim committed:
More fixes

GitHubber17 committed:
Merge pull request #37486 from CarlRabeler/master

jovan changes

PRMerger15 committed:
Merge pull request #37612 from sethmanheim/wk4-11

Fixing incorrect arm option

PRMerger16 committed:
Merge pull request #37613 from cynthn/files-fix

Fixing links and syntax

rwike77 committed:
adding container logs article

Ja-Dunn committed:
Merge pull request #36988 from v-viinde/ziflow

USER STORY 1226462 SaaS App Tutorial: Ziflow

GitHubber17 committed:
Merge pull request #37347 from v-nagta/digicert

USER STORY 1229999 SaaS App Tutorial: DigiCert Tutorial update

GitHubber17 committed:
Merge pull request #37470 from v-nagta/appnetamonitor

USER STORY 1224167 SaaS App Tutorial: AppNeta Performance Monitor

GitHubber17 committed:
Merge pull request #37543 from v-nagta/mindwireless

USER STORY 1226527 SaaS App Tutorial: mindWireless

GitHubber17 committed:
Merge pull request #37591 from cynthn/connect

Connect art

rmca14 committed:
Merge pull request #37452 from barbkess/gen2

reorganize tiers for gen2

liumichelle committed:
updated multiple subnet configuration recommendation for xrp migration

rwike77 committed:
updated text

liumichelle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into update-xRPMigration

updating the recommendation for multiple subnet configuration

ktoliver committed:
Merge pull request #37608 from Minewiskan/aas0411

Updates for sign in using MSA accounts.

rwike77 committed:
typo

PRMerger17 committed:
Merge pull request #37624 from liumichelle/update-xRPMigration

Update multiple subnet recomendation for xrp migration

PRMerger19 committed:
Merge pull request #37621 from mattbriggs/mdb_20180411_PKI2

adding trademark

SnehaGunda committed:
Image and metadata updates

barbkess committed:
update metadata

SnehaGunda committed:
Updating per PR feedback

barbkess committed:
updates

barbkess committed:
update

PRMerger10 committed:
Merge pull request #37619 from SaurabhSharma-MSFT/patch-1

(ForumsTeam) Modified the guid example MicrosoftDocs/azure-docs#6967

barbkess committed:
acrolinx fixes

barbkess committed:
acrolinx fixes

JeffGoldner committed:
withdraw Veeam

JeffGoldner committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

tamram committed:
Update storage-how-to-create-file-share.md

BryanTrach-MSFT committed:
(forumsteam) fixes MicrosoftDocs/azure-docs#7057

Updated link about slots. Old link does not list the limits for slots.

Commit:
Merge pull request #37631 from MicrosoftDocs/tamram-patch-2

Update storage-how-to-create-file-share.md

PRMerger12 committed:
Merge pull request #37636 from BryanTrach-MSFT/patch-15

(forumsteam) fixes MicrosoftDocs/azure-docs#7057

PRMerger12 committed:
Merge pull request #37632 from JeffGoldner/master

Withdraw veeam

Commit:
Merge pull request #37574 from cephalin/msi

upgrade to core 2.0

ktoliver committed:
Merge pull request #35609 from SnehaGunda/ASAQS

Stream analytics quickstarts

GitHubber17 committed:
Merge pull request #37400 from jeffgilb/49

Added start and stop articles

barbkess committed:
update metadata

barbkess committed:
acrolinx fixes

GitHubber17 committed:
Merge pull request #37610 from rolyon/rolyon-link-fix

[Azure RBAC] Fix links for new role-based-access-control folder

SnehaGunda committed:
removing key

v-jasowells committed:
Update php.md

v-jasowells committed:
Update csharp.md

v-jasowells committed:
Update ruby.md

v-jasowells committed:
Update python.md

v-jasowells committed:
Update csharp.md

v-jasowells committed:
Update ruby.md

v-jasowells committed:
Update python.md

v-jasowells committed:
Update php.md

genlin committed:
update typo

genlin committed:
update wordings

PRMerger12 committed:
Merge pull request #37640 from v-jaswel/master

Minor bug fixes.

ktoliver committed:
Fix note markdown

Fix note markdown

barbkess committed:
fix empty bullet

barbkess committed:
add layout tag

ktoliver committed:
Merge pull request #37637 from barbkess/april2

update metadata

PRMerger17 committed:
Merge pull request #37554 from genlin/master305

storage-how-to-use-files-windows.md

PRMerger19 committed:
Merge pull request #37639 from SnehaGunda/master

removing key

Commit:
updating topic field to conceptual

PRMerger20 committed:
Merge pull request #37646 from Blackmist/hacking

updating topic field to conceptual

Commit:
cleaning metadata

PRMerger10 committed:
Merge pull request #37647 from Blackmist/hacking

cleaning metadata

Commit:
metadata cleaning

PRMerger16 committed:
Merge pull request #37649 from Blackmist/hacking

metadata cleaning

Commit:
metadata changes

mrbullwinkle committed:
incorporate David's feedback

Commit:
Merge pull request #37650 from Blackmist/hacking

metadata changes

Commit:
metadata cleanup

Brenduns committed:
updating screenshot to match recent edit

PRMerger10 committed:
Merge pull request #37651 from Blackmist/hacking

metadata cleanup

Commit:
metadata updates

rothja committed:
Fixing links for new SP image

PRMerger16 committed:
Merge pull request #37260 from carolinacmoravia/patch-666

L15680: Extra blank space in reference

Commit:
cleaning metadata

PRMerger19 committed:
Merge pull request #37654 from rothja/patch-28

Fixing links for new SP image

PRMerger20 committed:
Merge pull request #37577 from cynthn/cynthn-patch-9

Update storage-about-vhds-and-disks-windows-and-linux.md

mrbullwinkle committed:
add analytics query link + note clarification

itechedit committed:
Merge pull request #37627 from barbkess/april

April

PRMerger14 committed:
Merge pull request #37446 from thedanfernandez/patch-1

Update Home.md

itechedit committed:
Merge pull request #37652 from Brenduns/april-12

updating screenshot to match recent edit

mrbullwinkle committed:
all titles must be sentence case fix

rwike77 committed:
added heading

itechedit committed:
Merge pull request #37653 from Blackmist/hacking

metadata updates

mrbullwinkle committed:
bulleted list spacing

mrbullwinkle committed:
sentence flow fix

mrbullwinkle committed:
fix

itechedit committed:
Merge pull request #37622 from rwike77/containerlogs

Container logs

v-alje committed:
Resolving merge conflict

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

itechedit committed:
Merge pull request #36590 from mrbullwinkle/mrb_03_29_2018_cohorts

Cohorts

v-alje committed:
Merge pull request #37659 from MicrosoftDocs/FromPublicRepo

From public repo
